app/com/m3/octoparts/repository/DBConfigsRepository.scala

```diff
@@ -57,7 +57,7 @@ trait ImmutableDBRepository extends ConfigsRepository {
   // Configs
   def findConfigByPartId(partId: String)(implicit parentSpan: Span): Future[Option[HttpPartConfig]] = {
     getWithSession(HttpPartConfigRepository, sqls.eq(HttpPartConfigRepository.defaultAlias.partId, partId), includes = Seq(HttpPartConfigRepository.hystrixConfigRef))
-      .trace(s"$zipkinSpanNameBase-findConfigByPartId:$partId")
+      .trace(s"$zipkinSpanNameBase-findConfigByPartId", "partId" -> partId)
```

Comment:
The s"" is needed here because I'm using `$zipkinSpanNameBase` in the string, but point taken. I've removed it from most of the places where it is unnecessary.